src/style.css

```diff
@@ -0,0 +1,91 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir.ttc);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir Heavy";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+}
+
+body {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  font-weight: 300;
+  margin-top: 20px;
+}
+
+.search-bar {
+  width: 100%;
+  position: relative;
+}
+
+.search-bar_input__big {
+  height: 70px;
+  width: 100%;
+  padding-left: 62px;
+  font-size: 16px;
+}
+
+.image-big {
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  top: 25px;
```

Comment:
Pay attention to your CSS properties order
https://9elements.com/css-rule-order/